buddyboss/buddyboss-platform
# PROD-844 - Fix #2243 : Past events not showing in search results

### Changes proposed in this Pull Request:



Fixes #2243

### How to test the changes in this Pull Request:

1. Need to have the BB demo DB to reproduce this issue. 
2. After installing the BB demo DB to your local instance, search for an event named "Songkran".
3. You will see all the events named with "Songkran" will be shown.

### Proof Screenshots or Video


https://www.loom.com/share/1f6f0d54ed8449bcb6ecf016aecdc461
